upgrading/extending-daml-models/0_intro.md

```diff
@@ -0,0 +1,18 @@
+Database schemas tend to evolve over time. Your business might change, the data that you collect
+might need to be extended or your access rights might need updating. For example, you might decide
+to interact with a new partner and share your database, but the access rights need to be different
+from what you have deployed for your other partners. Or maybe you decide to share parts of your data
+of a running ledger with the public for reading but not writing.
+
+In this first article on DAML ledger upgrades you learn how to extend your DAML model and ship this
+extension as a DAML package to a running ledger. This package will depend on the already deployed
+DAML packages on the ledger. As a running example, we will extend the social network you build in
+the [Getting Started Guide](https://daml.com/learn/getting-started/) with a forum.
+
+In the follow up article on DAML upgrades, you will learn how to make changes to an already deployed
+DAML package and how to migrate the data to your new model.
+
+You can find more documentation on upgrading DAML models and ledgers
+[here](https://docs.daml.com/upgrade/index.html).
+
+[![asciicast](https://asciinema.org/a/hl8tMxGU2raSodzX1VFTxYuTQ.svg)](https://asciinema.org/a/hl8tMxGU2raSodzX1VFTxYuTQ)
```

Comment:
I love ascicinema, but in our case it results in a user having to leave the page in order to see it (you can't play it on the scenario page, new tab opens up). Would really prefer a GIF

Reply:
I think there is a way to make a gif out of it, I'll see if I can do it.

Reply:
it's a gif now, thanks for recording it.